dashboards-observability/public/components/event_analytics/explorer/log_explorer.tsx

```diff
@@ -85,7 +86,7 @@ export const LogExplorer = ({
 
   const handleTabClick = (selectedTab: EuiTabbedContentTab) => {
     history.replace(
-      `/event_analytics/explorer/${queryRef.current![selectedTab.id][SAVED_OBJECT_ID] || ''}`
+      `/event_analytics/explorer/${queryRef.current![selectedTab.id][URL_SEARCH]}`
```

Comment:
I may need some help with manual regression testing... I think I covered most use cases here but not sure about all the workflows:
- app analytics uses this same component
- does this affect live tail?

Reply:
Not just here, but this feature overall